paper.tex

```diff
@@ -35,6 +35,29 @@
 \begin{document}
 \maketitle
 
+\abstract{
+  Symbolic based computer aided algebra systems (CAS) play an important role in
+  scientific computing with the earliest packages dating back into the 1970s.
+  Numerous open and proprietary software packages exist that aid users in
+  complex algebra, calculus, and other mathematical concepts that drive and
+  support both symbolic and numerical computing needs.
+  Here we present a symbolic CAS written in the Python programming language
+  that is developed by a large team of scientists, engineers, and software
+  developers in a bazaar open source development model.
```

Comment:
I don't know if I would use the term `bazaar` here. I don't even know what it means. I am sure I can look it up (e.g. I vaguely heard about "bazaar vs cathedral model", but I can't actually tell the difference off top of my head). So most readers will probably don't know what this term means either.